third_party/terraform/tests/resource_cloudfunctions_function_test.go

```diff
@@ -598,11 +599,11 @@ resource "google_cloudfunctions_function" "function" {
     // put code in it, so we created this repository outside the test to be used
     // here. If this test is run outside of CI, it may fail because of permissions
     // errors.
-    url = "https://source.developers.google.com/projects/hc-terraform-testing/repos/cloudfunctions-test-do-not-delete/moveable-aliases/master/paths/"
+    url = "https://source.developers.google.com/projects/%s/repos/cloudfunctions-test-do-not-delete/moveable-aliases/master/paths/"
```

Comment:
Will we recreate this repo in every project? I'd be curious if we could just open up the repo in `hc-terraform-testing`

Reply:
Nah, this doesn't actually create anything. It requires a repo to exist already with that name and a cloud function in it. I just created one for our test suite's project.
TBH - I'm not sure if permissions could make the hashi corp project open, but even if it's possible I'm not sure it's the right choice. It would mean they'd have to open it to the world and I think it's better that a user can just create a repo in their own GCP project.

Reply:
In that case, would you mind updating the comment to indicate that users will need to create their own repo instead of saying that it's already been created?